samples/drivers/counter/alarm/README.rst

```diff
@@ -8,6 +8,8 @@ Overview
 This sample provides an example of alarm application using counter API.
 It sets an alarm with an initial delay of 2 seconds. At each alarm
 expiry, a new alarm is configured with a delay multiplied by 2.
+Note: In case of 1Hz frequency (RTC for example), precision is 1 second.
+Now  value can be slightly different than the one in Sample Output
```

Comment:
Use RST notes, e.g.

```rst
.. note::
    In case of....
```

I'd suggest `Therefore, the sample output may differ in 1 second` instead of `Now  value can be slightly different than the one in Sample Output`

Reply:
dONE

Reply:
Docs build is broken, can you check?